healthcheck/healthcheck_test.go

```diff
@@ -62,7 +62,35 @@ func TestEvaluatingErrorsChecks(t *testing.T) {
 		log.Error("Healthchecks did not throw an error, but should have")
 		t.Fail()
 	}
-	if len(results) != 2 {
+	if len(err) != 3 {
+		log.Error("Healthcheck errors had the wrong length")
+		t.Fail()
+	}
+	if len(results) != 4 {
+		log.Error("Healthcheck results had the wrong length")
+		t.Fail()
+	}
+}
+
+func TestPreformingingErrorsChecks(t *testing.T) {
+	cxn := database.GetPGConnection(conf.DBURI())
+	healthChecks, _ := ReadHealthCheckYAMLFromFile("healthchecksErrors.yml")
+	results, HCerrs := healthChecks.PreformHealthChecks(cxn)
+
+	// log.Info("Spew healthcheck results")
```

Comment:
Can we either remove these commented out lines, or change them to log.Debug if they might be needed?